build_dist.sh

```diff
@@ -18,15 +18,11 @@ if ! grep -q "$LAST" doc/source/whats_new.rst ; then
   exit -1
 fi
 
-make cython
-
```

Comment:
Poetry will cythonize the Cython file automatically as implemented in `build.py`